src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -63,7 +63,7 @@ public function exists($name)
      *
      * @throws \Twig_Error_Loader if the template could not be found
      */
-    protected function findTemplate($template)
+    protected function findTemplate($template, $throw = true)
     {
         $logicalName = (string) $template;
```

Comment:
Would it not be enough to cast the template reference to string before calling the twig loader methods in this place: https://github.com/symfony/symfony/blob/2.3/src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php#L122

Ref. #12197